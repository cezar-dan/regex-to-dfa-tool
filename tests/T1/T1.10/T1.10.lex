 
SPACE
0
0,' ',1
1,' ',2
1

\n
NEWLINE
0
0,'\n',1
1,'\n',2
1

abcdef
PATTERN1
0
0,'a',1
0,'b',2
0,'c',1
0,'d',1
0,'e',3
0,'f',1
2,'a',5
2,'b',4
2,'c',6
2,'d',1
2,'e',7
2,'f',1
3,'a',5
3,'b',4
3,'c',6
3,'d',1
3,'e',7
3,'f',1
4,'a',5
4,'b',4
4,'c',6
4,'d',1
4,'e',7
4,'f',1
5,'a',5
5,'b',2
5,'c',6
5,'d',1
5,'e',7
5,'f',1
6,'a',11
6,'b',1
6,'c',6
6,'d',1
6,'e',1
6,'f',1
7,'a',5
7,'b',4
7,'c',6
7,'d',1
7,'e',7
7,'f',8
8,'a',1
8,'b',1
8,'c',1
8,'d',9
8,'e',1
8,'f',1
9,'a',1
9,'b',1
9,'c',1
9,'d',1
9,'e',10
9,'f',1
10,'a',1
10,'b',1
10,'c',1
10,'d',1
10,'e',10
10,'f',8
11,'a',11
11,'b',1
11,'c',6
11,'d',1
11,'e',1
11,'f',1
3 9 4 7 11 6 5 2

abcd
PATTERN2
0
0,'a',1
0,'b',2
0,'c',3
0,'d',4
1,'a',1
1,'b',2
1,'c',3
1,'d',14
2,'a',1
2,'b',2
2,'c',3
2,'d',14
4,'a',5
4,'b',6
4,'c',7
4,'d',8
5,'a',3
5,'b',3
5,'c',3
5,'d',10
6,'a',3
6,'b',3
6,'c',3
6,'d',8
7,'a',1
7,'b',2
7,'c',3
7,'d',11
8,'a',5
8,'b',6
8,'c',9
8,'d',8
9,'a',3
9,'b',3
9,'c',3
9,'d',12
10,'a',5
10,'b',3
10,'c',9
10,'d',3
11,'a',3
11,'b',3
11,'c',7
11,'d',3
12,'a',3
12,'b',3
12,'c',9
12,'d',3
13,'a',1
13,'b',2
13,'c',3
13,'d',14
14,'a',3
14,'b',3
14,'c',13
14,'d',3
7 8 6 9 5 4 13 2 1 0

bdef
PATTERN3
0
0,'b',1
0,'d',2
0,'e',3
0,'f',1
2,'b',7
2,'d',1
2,'e',1
2,'f',1
3,'b',1
3,'d',2
3,'e',6
3,'f',1
4,'b',1
4,'d',2
4,'e',4
4,'f',5
5,'b',1
5,'d',2
5,'e',3
5,'f',5
6,'b',1
6,'d',2
6,'e',4
6,'f',5
7,'b',1
7,'d',2
7,'e',3
7,'f',1
5 4 6 3 7

adef
PATTERN4
0
0,'a',1
0,'d',2
0,'e',3
0,'f',4
1,'a',1
1,'d',2
1,'e',3
1,'f',4
2,'a',6
2,'d',2
2,'e',3
2,'f',4
3,'a',9
3,'d',10
3,'e',9
3,'f',9
4,'a',7
4,'d',9
4,'e',9
4,'f',4
5,'a',1
5,'d',2
5,'e',3
5,'f',5
6,'a',1
6,'d',2
6,'e',3
6,'f',5
7,'a',1
7,'d',2
7,'e',3
7,'f',4
8,'a',1
8,'d',2
8,'e',3
8,'f',5
10,'a',11
10,'d',9
10,'e',9
10,'f',9
11,'a',9
11,'d',9
11,'e',9
11,'f',8
7 5 8 6 2 1

abcdf
PATTERN5
0
0,'a',1
0,'b',2
0,'c',3
0,'d',4
0,'f',5
1,'a',1
1,'b',2
1,'c',3
1,'d',4
1,'f',5
2,'a',1
2,'b',2
2,'c',3
2,'d',4
2,'f',5
3,'a',1
3,'b',2
3,'c',3
3,'d',4
3,'f',5
4,'a',1
4,'b',2
4,'c',3
4,'d',4
4,'f',5
5,'a',1
5,'b',2
5,'c',3
5,'d',4
5,'f',5
5 4 3 2 1 0