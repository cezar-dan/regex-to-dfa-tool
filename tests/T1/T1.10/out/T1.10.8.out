PATTERN3 eedbeeeeeedbeefeeefffdbdb
SPACE  
PATTERN5 dbafcacdcfffdfbdcfbfccdad
SPACE  
PATTERN1 ebbbacacacaccaaaacccaaaca
NEWLINE \n